### ISSUE TITLE
Model.query() does not return new mquery() instance.

### ISSUE THREAD

**Issue by mikejholly** [modella/mongo#8]
In mquery.js...

``` js
module.exports = function(Model, collection) {
  var queryWrapper = buildQueryWrapper(Model, collection);

  Model.query = function() {
    return queryWrapper;
  };
};
```

`Model.query()` will return the same instance of mquery on every invocation. This leads to unexpected results when using `Model.query()` after a `model.save()` call. For example, it looks like the `op` and `_conditions` properties are set on the result of `User.query()` after I call `save()` on a user instance, as if the query is maintaining it's state. 

Would it not be preferable to simply instantiate a new reference to mquery for each `query()` invocation? 

Something like:

``` js
module.exports = function(Model, collection) {
  Model.query = function() {
    return buildQueryWrapper(Model, collection);
  };
};
```

**Comment by mikejholly** [comment]
Actually, the `model.save` call is not related--it's just multiple calls to `User.query()` that's causing the problem.

**Comment by rschmukler** [comment]
Thanks for the find. I fixed this and published under `0.2.8` now available on NPM.
